src/base/ClpMap.h

```diff
@@ -17,18 +17,24 @@
 #include <list>
 #include <unordered_map>
 
-template<class EntryValue>
+template<class Value>
 size_t
-DefaultMemoryUsage(const EntryValue *e)
+DefaultMemoryUsage(const Value &e)
 {
-    return sizeof(*e);
+    return sizeof(e);
 }
 
-/// An in-memory cache enforcing three primary policies:
-/// Capacity: The memory used by cached entries has a configurable limit;
-/// Lifetime: Entries are hidden (and may be deleted) after their TTL expires;
-/// Priority: Capacity victims are purged in LRU order.
-template <class Key, class EntryValue, size_t MemoryUsedByEV(const EntryValue *) = DefaultMemoryUsage>
+/// An in-RAM associative container enforcing three primary caching policies:
```

Comment:
"in-RAM" assumes that RAM is the only form of memory. This is becoming less and less true for modern machines. If you want to be specific this had better be "in-NAND" or "in-crystal".